### PR TITLE
update event model to typescript

### PR DESCRIPTION
**Pull request by Pespiri**

### CODE REVIEW

File: src/components/events-list/event-summary.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ const EventSummary = ({ event }) => (
 );
 
 EventSummary.propTypes = {
-  event: PropTypes.shape(eventModel).isRequired,
+  event: PropTypes.shape(EventModelValidationMap).isRequired,
 };
 
 export default EventSummary;
```
Comment (nilsgstrabo) [inline src/components/events-list/event-summary.js:56]: The default export is not used anywhere and can be removed

File: src/components/events-list/index.js [hunk 2]
```diff
@@ -45,7 +45,8 @@ export const EventsList = ({ events }) => (
 );
 
 EventsList.propTypes = {
-  events: PropTypes.arrayOf(PropTypes.shape(eventModel)).isRequired,
+  events: PropTypes.arrayOf(PropTypes.shape(EventModelValidationMap))
+    .isRequired,
 };
 
 export default EventsList;
```
Comment (nilsgstrabo) [inline src/components/events-list/index.js:52]: The default export is not used anywhere and can be removed

**Review by nilsgstrabo — APPROVED** [review]
Approved. Just a couple of default exports that are no longer used, and should probably be removed.

File: src/components/events-list/index.js (hunk 2 above)
Comment (Pespiri) [reply src/components/events-list/index.js]: Thanks for the observation. I will have this fixed

File: src/components/events-list/event-summary.js (hunk 1 above)
Comment (Pespiri) [reply src/components/events-list/event-summary.js]: Thanks for the observation. I will have this fixed

**Review by nilsgstrabo — APPROVED** [review]

**Pull request merged**